pages/installing-slimefun.md

```diff
@@ -0,0 +1,45 @@
+# Installing Slimefun
+
```

Comment:
BTW you should rename this file to "Installing-Slimefun", wiki files should use capital letters at the beginning of words.